### PR TITLE
doc(kit): Initial release for Manufacturing as a Service Kit

### PR DESCRIPTION
**Pull request by dominikoeh**
## Description

This contribution contains the initial release for the Manufacturing as a Service Kit from the Resiliency Domain

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+The reference architecture includes standardized and tested interfaces and data models that can be used for the development and implementation of Manufacturing-as-a-Service solutions (Chapter 4 of Whitepaper).
+
+### Standards
+
+Standardizing the necessary interfaces and data models of our architecture enables to exchange data with different systems and users and to extend the reference architecture with additional services, replace certain individual services with equivalent ones or to remove optional services.
+
+Within the Resiliency KiT Manufacturing as a service, two main standards were defined:
+
+- Manufacturing Capability Model and API - for declaration of offered and required manufacturing capabilities
+- Request for Quotation Model and API - for manufacturing demand configuration
```
Comment (maximilianong) [inline docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md:29]: Can you link to the standards or if they are not published - please provide the number, the name and the relevant version and link to the standard library. 

In addition the standards should be h2 and one of the last points on the adoption view.

**Comment by maximilianong** [comment]
**Overall:** 

Very technical adoption view.

Standards should be added.

Will the API come with 24.03?
If not I would recommend to publish this KIT under "upcoming KITs" https://eclipse-tractusx.github.io/docs-kits/kits/Resiliency/

File: docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md (hunk 1 above)
Comment (dominikoeh) [reply docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md]: Section has been moved and links/numbers have been provided

File: docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+id: Adoption View MaaS Kit
```
[the remark this answers is not in the document]
Comment (dominikoeh) [reply docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md]: KIT naming has been adopted throughout the release

**Review by maximilianong — APPROVED** [review]
Thanks @dominikoeh for providing the changes! 
Ready to go for R 24.03

**Comment by danielmiehle** [comment]
@dominikoeh can you please have a look at the merge conflict? we would like to merge today. Thanks!

**Comment by dominikoeh** [comment]
@danielmiehle @maximilianong merge conflict has been resolved

**Review by danielmiehle — APPROVED** [review]

**Comment by maximilianong** [comment]
@SebastianBezold / @Siegfriedk can someone of you support here?

@dominikoeh hast the problem re-produce this error locally 
 java -jar dash.jar package-lock.json -summary DEPENDENCIES

**Review by SebastianBezold — APPROVED** [review]

**Comment by SebastianBezold** [comment]
> @SebastianBezold / @Siegfriedk can someone of you support here?
> 
> @dominikoeh hast the problem re-produce this error locally java -jar dash.jar package-lock.json -summary DEPENDENCIES

triggered the check again. No issues now. re-triggering can be done by every committer

**Comment by maximilianong** [comment]
> > @SebastianBezold / @Siegfriedk can someone of you support here?
> > @dominikoeh hast the problem re-produce this error locally java -jar dash.jar package-lock.json -summary DEPENDENCIES
> 
> triggered the check again. No issues now. re-triggering can be done by every committer

Easy, thanks
Like turning the WiFi router off and on again.

**Pull request merged**